apps/frontend/src/components/xpress/XpressProposalTable.tsx

```diff
@@ -0,0 +1,430 @@
+import { useDataApi } from 'hooks/common/useDataApi';
+import { useProposalStatusesData } from 'hooks/settings/useProposalStatusesData';
+import { useTechniquesData } from 'hooks/technique/useTechniquesData';
+import { StyledContainer, StyledPaper } from 'styles/StyledComponents';
+import {
+  addColumns,
+  fromArrayToCommaSeparated,
+  setSortDirectionOnSortField,
+} from 'utils/helperFunctions';
+import { tableIcons } from 'utils/materialIcons';
+
+import { ProposalViewData } from './useProposalsCoreData';
+import { useXpressInstrumentsData } from './useXpressInstrumentsData';
+import XpressProposalFilterBar from './XpressProposalFilterBar';
+
+const XpressProposalTable = () => {
+  const tableRef = React.useRef<MaterialTableCore<ProposalViewData>>();
+  const refreshTableData = () => {
+    tableRef.current?.onQueryChange({});
+  };
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [selectedProposals, setSelectedProposals] = useState<
+    ProposalViewData[]
+  >([]);
+  const { techniques, loadingTechniques } = useTechniquesData();
+  const { proposalStatuses, loadingProposalStatuses } =
+    useProposalStatusesData();
+
+  const [searchParams, setSearchParams] = useSearchParams({});
+
+  const callId = searchParams.get('callId');
+  const instrument = searchParams.get('instrument');
+  const technique = searchParams.get('technique');
+  const proposalId = searchParams.get('proposalId');
+  const to = searchParams.get('to');
+  const from = searchParams.get('from');
+  const proposalStatusId = searchParams.get('proposalStatusId');
+  const search = searchParams.get('search');
+  const selection = searchParams.getAll('selection');
+  const sortField = searchParams.get('sortField');
+  const sortDirection = searchParams.get('sortDirection');
+  const page = searchParams.get('page');
+  const pageSize = searchParams.get('pageSize');
+
+  const api = useDataApi();
+  const { currentRole } = useContext(UserContext);
+
+  const [proposalFilter, setProposalFilter] = useState<ProposalsFilter>({
+    callId: callId ? +callId : undefined,
+    instrumentFilter: {
+      instrumentId: instrument ? +instrument : null,
+      showAllProposals: !instrument,
+      showMultiInstrumentProposals: false,
+    },
+    techniqueFilter: {
+      techniqueId: technique ? +technique : null,
+      showAllProposals: !technique,
+      showMultiTechniqueProposals: false,
+    },
+    dateFilter: {
+      to: to ? to : null,
+      from: from ? from : null,
+    },
+    referenceNumbers: proposalId ? [proposalId] : undefined,
+    proposalStatusId: proposalStatusId ? +proposalStatusId : undefined,
+    text: search,
+    excludeProposalStatusIds: [9],
+  });
+
+  useEffect(() => {
+    let isMounted = true;
+
+    if (isMounted) {
+      refreshTableData();
+    }
+
+    return () => {
+      isMounted = false;
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [JSON.stringify(proposalFilter)]);
```

Comment:
We can also create a function that we pass to XpressProposalFilterBar component that will update our setProposalFilter and update the table.